Marlin/src/lcd/menu/menu_configuration.cpp

```diff
@@ -354,6 +354,9 @@ void menu_advanced_settings();
 
   #include "../../feature/controllerfan.h"
 
+namespace LanguageNarrow_en {
+  static LSTR MSG_E_FAN = _UxGT("Extruder fan speed");
```

Comment:
This should be in `language_en.h` and not directly defined here. This will allow text to be translated to other languages.

Reply:
> This should be in `language_en.h`

Sure, I just tried to make quick check whether it's working at all.